### ISSUE TITLE
Question to oauth2  - invalid_scope 

### ISSUE THREAD

**Issue by AndroidJVCDER** [AndlyticsProject/andlytics#682]
Hallo everybody :-)

I tried to fork your project to use some parts of the code.Currently I have a new packagename in my project.
I got an" com.google.android.gms.auth.GoogleAuthException: INVALID_SCOPE" exception
 here:

oauthLoginToken = GoogleAuthUtil.getTokenWithNotification(AndlyticsApp.getInstance(), accountName, OAUTH_LOGIN_SCOPE, null); 

If I use the andlytics packagename everything goes right.

Do I have to add something to the developer console or even do somehting special with apk signing?

Thank you so much :-)

**Comment by nelenkov** [comment]
I think you need to create a new client ID for your application in Google's developer console. You need to use your own package name and signing certificate hash. 

https://console.developers.google.com

**Comment by AndroidJVCDER** [comment]
thank you for answering:) 

I actually did, but there is no permission for the scope: oauth2:https://www.google.com/accounts/OAuthLogin  

What kind of config, permission should I add there?

**Comment by nelenkov** [comment]
Not sure what you are doing, but the only two fields in the 'Create Client ID' dialog are 'package name' and 'signing certificate fingerprint'. You don't need any permissions.

**Comment by AndroidJVCDER** [comment]
I exactly did this, but always invalid scope on different devices... :-( 
Did you create two client ids, one for debug and one for release? I asked myself, why I could get a token with andlytics code without signing with your keystore or even if I used the packagename with totally different signing. Is this an security issue?!

Maybe someone has an idea how solve this?
I am confused..

**Comment by nelenkov** [comment]
Yes, there are debug and release Client IDs. `oauth2:https://www.google.com/accounts/OAuthLogin` is not a 'real' scope, it just gives you a token so you can access other services, so you shouldn't need to register. Did you change anything else?

**Comment by AndroidJVCDER** [comment]
Can we get in private contact? I would like to pay you for your help :-)
